src/three-render-objects.js

```diff
@@ -298,7 +306,23 @@ export default Kapsule({
     }
 
     state.renderer.setSize(state.width, state.height);
-    state.camera.aspect = state.width/state.height;
+
+    if (state._cameraType === 'perspective') {
+      state.camera.aspect = state.width/state.height;
+    } else {
```

Comment:
Does all of this code need to be run at initialization? It seems fully duplicated from the component update part so perhaps having it there is sufficient?